Build fails

```sh
export CC="gcc"
export CFLAGS="-DWOLFSSL_AES_DIRECT -DWOLFSSL_STATIC_PSK -DHAVE_AES_ECB -DWOLFSSL_DES_ECB -DHAVE_ECC_SECPR2 -DHAVE_ECC_SECPR3 -DHAVE_ECC_BRAINPOOL -DHAVE_ECC_KOBLITZ"

git clone --depth 1 https://github.com/wolfSSL/wolfssl.git
git clone --depth 1 https://github.com/wolfSSL/wolfssh.git

cd wolfssl/
autoreconf -ivf
./configure --enable-static --disable-examples --disable-crypttests --disable-asm --enable-tls13 --enable-ocsp --enable-dtls --enable-sni --enable-blake2 --enable-blake2s --enable-curve25519 --enable-session-ticket --enable-nullcipher --enable-crl --enable-ed25519 --enable-psk --enable-earlydata --enable-postauth --enable-hrrcookie --enable-opensslextra --enable-certext --enable-tlsx --enable-oldtls --enable-tlsv10 --enable-indef --enable-psk --enable-ecccustcurves=all --enable-secure-renegotiation  --enable-curve25519 --enable-curve448 --enable-ed25519 --enable-ed448 --enable-ocspstapling --enable-srp --enable-ssh --enable-keygen
make -j$(nproc)
export LDFLAGS="-L$(realpath src/.libs/)"
export CFLAGS="$CFLAGS -I $(realpath .)"
cd ../

cd wolfssh/
autoreconf -ivf
./configure --enable-static --enable-all --disable-examples
make -j$(nproc)
```

This fails with the following error:

```
examples/client/client.c:1088:17: error: ‘IPOverride’ undeclared (first use in this function)
 1088 |                 IPOverride = 1;
      |                 ^~~~~~~~~~
```

--- a/examples/client/client.c
+++ b/examples/client/client.c
@@ -1084,10 +1084,11 @@ THREAD_RETURN WOLFSSH_THREAD client_test(void* args)
                 caCert = myoptarg;
                 break;
 
+            #if defined(OPENSSL_ALL) || defined(WOLFSSL_IP_ALT_NAME)
             case 'X':
                 IPOverride = 1;
                 break;
-
+            #endif
         #endif
 
             case 'x':

--- a/examples/sftpclient/sftpclient.c
+++ b/examples/sftpclient/sftpclient.c
@@ -1675,10 +1675,11 @@ THREAD_RETURN WOLFSSH_THREAD sftpclient_test(void* args)
                 caCert = myoptarg;
                 break;
 
+            #if defined(OPENSSL_ALL) || defined(WOLFSSL_IP_ALT_NAME)
             case 'X':
                 IPOverride = 1;
                 break;
-
+            #endif
         #endif
 
             case '?':

